erdpy/wallet/keyfile.py

```diff
@@ -65,6 +69,64 @@ def load_from_key_file(key_file_json, password):
     return address_bech32, seed
 
 
+def save_to_key_file(json_path: Path, seed: str, pubkey: str, password: str) -> None:
+    address = accounts.Address(pubkey)
+    address_hex = address.hex()
+
+    address_bech32 = address.bech32()
+
+    pvt_key = seed + pubkey
```

Comment:
Warning: `save_to_key_file()` is always passed a true private key in the `seed` parameter when called. This concatenation is unnecessary.

Please use this implementation as reference: https://github.com/ElrondNetwork/elrond-core-js/blob/200c376b0709880f581478cbd20457089f063276/src/account.js#L76

Reply:
Removed the concatenation